onnxruntime/core/graph/model.cc

```diff
@@ -703,6 +731,13 @@ common::Status Model::LoadFromOrtFormat(const fbs::Model& fbs_model,
       schema_registry->RegisterRegistry(schema_collection);
     }
   }
+
+  // Populate the metadata to model_proto
```

Comment:
Not sure of this part is necessary, only thing I can think of is to convert an ort model back to onnx format, maybe we can remove this to save a bit binary size.